tensorflow/training/docker/2.4/py3/cu110/Dockerfile.gpu

```diff
@@ -218,6 +230,11 @@ RUN cat /etc/ssh/ssh_config | grep -v StrictHostKeyChecking > /etc/ssh/ssh_confi
 # Add NGC vars
 ENV TF_AUTOTUNE_THRESHOLD=2
 
+RUN SMDATAPARALLEL_TF=1 pip install --no-cache-dir ${SMDATAPARALLEL_BINARY} \
+ && rm -rf ${SMDATAPARALLEL_BINARY}
```

Comment:
If you `pip install` from an HTTPS link, you don't need to `rm -rf` it after the installation, because the file has not actually been downloaded into this folder.